ervinu/web content/style.css

```diff
@@ -0,0 +1,105 @@
+body {
+    background-color: rgb(211, 207, 207);
+    margin: 10px 5px;
+    font-family: sans-serif;
+}
+
+#top-container {
+    background-color: #ffffff;
+    margin: 5px;
+}
+
+#title h2, #title a {
+    display: inline-block;
+    margin: 5px 10px;
+}
+
+a {
+text-decoration: none;
+}
+
+.carousel {
+    background-color: rgb(255, 255, 255);
+    height: 330px;
+    width: 99%;
+    position: relative;
+    display: flex;
+}
+
+.carousel p {
+    margin: 4px 0px;
+}
+
+.carousel img {
+    background-color: rgb(236, 236, 236);
+    height: 70%;
+    width: 98%;
+    margin: 0px 10px;
+}
+
+
+.deal-discount p:first-child {
+    background-color: rgb(255, 0, 0);
+    color: rgb(255, 255, 255);
+    display: inline-block;
+    margin-left: 10px;
+}
+
+.deal-discount p:last-child {
+    color: red;
+    display: inline-block;
+}
+
+.euro-sign {
+    font-size: 0.5em;
+    position: relative;
+    top: -0.25em;
```

Comment:
don't use em, use % or px

Reply:
If I don't use top: -5px; for example my euro sign is positioned at the bottom of number 2